app/actions/StatisticsActions.js

```diff
@@ -391,7 +390,7 @@ export const exportStatToCSV = (opts) => (dispatch, getState) => {
   // called once for each data line
   const progressFunction = (time, series) => {
     const values = allSeries.map((s) =>
-      !isUndefined(series[s.name]) ? s.valueFormatFunc(series[s.name]) : null
+      series[s.name] !== undefined ? s.valueFormatFunc(series[s.name]) : null
```

Comment:
This isn't enough, you could also check if typeof x === "undefined" but I would rather user lodash `isUndefined` func for that.

Lodash may have it's own version of `isNumber, isNullOrUndefined` funcs as well, checkout their documentation.